### PR TITLE
Google Cloud Storage support

### PR DESCRIPTION
**Pull request by littleq0903**
manual QA passed.

now adding unit testing on it.

### CODE REVIEW

**Comment by winhamwr** [comment]
Hi Colin,

It also looks like you have some flake8 issues to clear up, if you didn't notice that build fail.

-Wes

**Comment by littleq0903** [comment]
I checked with PyFlake from my side, didn't see any error in the code I added, but found two additional errors in the beginning of tester classes, now fixed.

but from the message it seems like the error refers to the command-level error? if pep8 error it should point out which line.

**Comment by littleq0903** [comment]
oh wait, I checked the wrong files. let me fix it.

**Comment by littleq0903** [comment]
figured it out, I recently made my PyFlake vim plugin to ignore E501 errors (line too long exceed 80 chars per line), fixed all lines exceed the 80 chars.

**Comment by littleq0903** [comment]
now this left:

`ERROR: InvocationError: '/home/travis/build/PolicyStat/terrarium/.tox/py26-venv-1.8.4/bin/nosetests -v'`

File: terrarium/terrarium.py [hunk 1]
```diff
@@ -760,6 +844,48 @@ def parse_args():
...
+        '--gcs-project',
+        default=os.environ.get('GCS_PROJECT', None),
+        help='''
+            Google Cloud Storage project.
+            Defaults to GCS_PROJECT env variable.
+        '''
+    )
+    ap.add_argument(
+        '--gcs-client-email',
+        default=os.environ.get('GCS_CLINET_EMAIL', None),
```
Comment (winhamwr) [inline terrarium/terrarium.py]: Typo here. `GCS_CLIENT_EMAIL`

**Comment by winhamwr** [comment]
Other than the typo, I think this needs:

* A unittest like [this one](https://github.com/PolicyStat/terrarium/blob/master/tests/tests.py#L517) that verifies the error message/code if GCS isn't installed
* GCS documentation that mirrors [the s3 docs](https://github.com/PolicyStat/terrarium/blob/master/docs/user_guide.rst#storing-terrarium-environments-on-s3) to let folks know they can use GCS.

It looks like your last failing travis build might have been just intermittent. I made a small change to master just to confirm that the travis build is passing there and it is.

**Comment by littleq0903** [comment]
@winhamwr : can you re-run the build#199 on Travis?

**Comment by winhamwr** [comment]
Hi Colin, 

Maybe fix the typo and push? I'm on my mobile and don't see an obvious way to do that. 

-Wes

**Comment by littleq0903** [comment]
docs added.

**Comment by winhamwr** [comment]
Excellent. Thanks, Colin!

**Pull request merged**